lib/src/utils/cache_helpers.dart

```diff
@@ -288,6 +289,7 @@ extension CacheUpdates on NyxxRest {
         EntitlementCreateEvent(:final entitlement) => updateCacheWith(entitlement),
         EntitlementUpdateEvent(:final entitlement) => updateCacheWith(entitlement),
         EntitlementDeleteEvent(:final entitlement) => entitlement.manager.cache.remove(entitlement.id),
+        SoundboardSound(:final user) => updateCacheWith(user),
```

Comment:
The other new events need adding here too (as no-ops) or a warning for unhandled events will be thrown.